src/pages/tutor-home/TutorHome.tsx

```
@@ -2,16 +2,19 @@ import { useEffect } from 'react'
 
 import { useAppSelector } from '~/hooks/use-redux'
 import { useModalContext } from '~/context/modal-context'
+import { Container, Stack } from '@mui/material'
 
 import UserStepsWrapper from '~/components/user-steps-wrapper/UserStepsWrapper'
 import FindBlock from '~/components/find-block/FindBlock'
 
-import { styles } from '~/pages/tutor-home/TutorHome.styles'
 import { translationKey } from '~/components/find-block/find-student-constants'
 import { HowItWorksBlock } from '~/components/how-it-works/HowItWorksBlock'
 import { Container, Stack } from '@mui/material'
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Container'. Consider to delete it or rename it.

'Container' is defined here:


(lint/suspicious/noRedeclare)